bugalink-frontend/pages/request/[id]/accept.tsx

```diff
@@ -79,14 +101,8 @@ export default function AcceptRequest() {
         )}
         <TripDetails
           date={dateText}
-          originHour={startTime.toLocaleTimeString('es-ES', {
-            hour: '2-digit',
-            minute: '2-digit',
-          })}
-          destinationHour={endTime.toLocaleTimeString('es-ES', {
-            hour: '2-digit',
-            minute: '2-digit',
-          })}
+          originHour={startTime}
+          destinationHour={endTime.substr(11,12)}
```

Comment:
This looks like a workaround that would likely fail for other locales (for example, if my phone is in english and yours is in spanish) as it relies on the string representation of the time